remove useless overhead

tpo opened the pull request:
##### SUMMARY

I'm unable to see why one would write:

    - name: Install a list of packages
      apt:
        name: "{{ packages }}"
      vars:
        packages:
        - foo
        - foo-tools

Instead of the simpler:

    - name: Install a list of packages
      apt:
        packages:
        - foo
        - foo-tools

So let's do just that, remove the mental and runtime overhead and use the simple form.
 
##### ISSUE TYPE
- Docs Pull Request

+label: docsite_pr

##### SUMMARY
<!--- Describe the change below, including rationale and design decisions -->

<!--- HINT: Include "Fixes #nnn" if you are fixing an existing issue -->

##### ISSUE TYPE
<!--- Pick one below and delete the rest -->
- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### COMPONENT NAME
<!--- Write the short name of the module, plugin, task or feature below -->

##### ADDITIONAL INFORMATION
<!--- Include additional information to help people understand the change here -->
<!--- A step-by-step reproduction of the problem is helpful if there is no related issue -->

<!--- Paste verbatim command output below, e.g. before and after your change -->
```paste below

```

tpo committed:
remove useless overhead

##### SUMMARY

I'm unable to why one would write:

    - name: Install a list of packages
      apt:
        name: "{{ packages }}"
      vars:
        packages:
        - foo
        - foo-tools

Instead of the simpler:

    - name: Install a list of packages
      apt:
        packages:
        - foo
        - foo-tools

So let's do just that, remove the mental and runtime overhead and use the simple form.
 
##### ISSUE TYPE
- Docs Pull Request

+label: docsite_pr

bcoca commented on lib/ansible/modules/packaging/os/apt.py:
```diff
@@ -168,8 +168,6 @@
 - name: Install a list of packages
   apt:
-    name: "{{ packages }}"
-  vars:
```
`packages` is not a valid option for `apt`, it is either `name`, `package` or `pkg`

tpo committed:
Fix apt module option name in example

as requested here: https://github.com/ansible/ansible/pull/59517#pullrequestreview-266015861
